src/mscorlib/ref/mscorlib.cs

```diff
@@ -15,6 +15,31 @@ public static partial class EnvironmentAugments
         public static void FailFast(string message, System.Exception error) { }
         public static string[] GetCommandLineArgs() { throw null; }
     }
+
+    public static partial class AppDomainAugments
+    {
+        public static int GetCurrentThreadId() { throw null; }
+        public static event System.AssemblyLoadEventHandler AssemblyLoad { add { } remove { } }
+        public static event System.ResolveEventHandler ResourceResolve { add { } remove { } }
+        public static event System.ResolveEventHandler TypeResolve { add { } remove { } }
+        public static event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException { add { } remove { } }
+        public static event System.EventHandler ProcessExit { add { } remove { } }
+        public static string ApplyPolicy(string assemblyName) { throw null; } 
+        public static int ExecuteAssembly(string assemblyFile, string[] args, byte[] hashValue, System.Configuration.Assemblies.AssemblyHashAlgorithm hashAlgorithm) { throw null; }
+        public static int ExecuteAssemblyByName(System.Reflection.AssemblyName assemblyName, params string[] args) { throw null; }
+        public static int ExecuteAssemblyByName(string assemblyName, params string[] args) { throw null; }
+        public static void SetData(string name, object data) { throw null; }
+        public static System.Nullable<bool> IsCompatibilitySwitchSet(string value) { throw null; }
+        public static System.Reflection.Assembly Load(byte[] rawAssembly) { throw null; } 
+        public static System.Reflection.Assembly Load(byte[] rawAssembly, byte[] rawSymbolStore) { throw null; }
+        public static System.Reflection.Assembly Load(System.Reflection.AssemblyName assemblyRef) { throw null; }
+        public static System.Reflection.Assembly Load(string assemblyString) { throw null; }
+        public static bool MonitoringIsEnabled { get { throw null; } set { } }
```

Comment:
@Maoni0 - Do you think that the AppDomain resource monitoring makes sense on CoreCLR without just a single appdomain?

My inclination would be to just throw from it on CoreCLR - same as Mono (https://github.com/mono/mono/blob/b190f213a364a2793cc573e1bd9fae8be72296e4/mcs/class/corlib/System/AppDomain.cs#L1650). If nothing else, it is hard to test that it work properly - if somebody actually used it for anything, it is unlikely to work anyway.